src/com/skillsup/dao/ProductDao.java

```diff
@@ -12,10 +10,11 @@
     void save(Product product);
     void update(Product product);
     void delete(Product product);
-    Product get(String productName);
+    Product getByName(String productName);
+    Product getById(long productName);
```

Comment:
ById(long productName)

Reply:
Yes, you are absolutely right! This is my carelessness. Of course here must be "long productId", not "productName")